types/tx_test.go

```diff
@@ -77,16 +77,40 @@ func TestUnwrapMalleatedTx(t *testing.T) {
 
 	// due to protobuf not actually requiring type compatibility
 	// we need to make sure that there is some check
-	_, ok = UnwrapMalleatedTx(rawBlock)
+	_, ok = UnmarshalMalleatedTx(rawBlock)
 	require.False(t, ok)
 
-	pHash := sha256.Sum256(rawBlock)
-	MalleatedTx, err := WrapMalleatedTx(pHash[:], 0, rawBlock)
+	MalleatedTx, err := MarshalMalleatedTx(0, rawBlock)
 	require.NoError(t, err)
 
 	// finally, ensure that the unwrapped bytes are identical to the input
-	malleated, ok := UnwrapMalleatedTx(MalleatedTx)
+	malleated, ok := UnmarshalMalleatedTx(MalleatedTx)
 	require.True(t, ok)
-	assert.Equal(t, 32, len(malleated.OriginalTxHash))
 	require.Equal(t, rawBlock, malleated.Tx)
 }
+
+func TestWrapUnmarshalBlobTx(t *testing.T) {
```

Comment:
[nit] the rest of this PR adopted marshal / unmarshal so it seems appropriate to remove "Wrap" from this test name

Reply:
damn... good catch!! will go through and double check names in a follow up PR

Reply:
https://github.com/celestiaorg/celestia-core/pull/905/files#r1043960666